hw/dv/sv/sim_sram/index.md

```diff
@@ -0,0 +1,80 @@
+## Simulation SRAM
+
+The `sim_sram` module intercepts an outbound TL interface to carve out a chunk of "fake" memory used for simulation purposes only.
+This chunk of memory must not overlap with any device on the system address map - it must be an invalid address range from the system's perspective.
+
+It has 4 interfaces - clock, reset, input TL interface and the output TL interface.
+It instantiates a [1:2 TL socket]({{< relref "hw/ip/tlul/doc#bus-primitives" >}}) to split the incoming TL access into two.
+One of the outputs of the socket is sinked within the module while the other is returned back.
+
+If the user chooses to instantiate an actual SRAM instance, the sinked TL interface is connected to TL SRAM adapter which converts the TL access into an SRAM access.
+The [technology-independent]({{< relref "hw/ip/prim/README" >}}) `prim_ram_1p` module is used for instantiating the memory.
+
+![Block Diagram](sim_sram.svg)
+
+This module is not meant to be synthesized.
+Though, it is written with the synthesizable subset of SystemVerilog to be Verilator-friendly.
+An easy way to ensure this is to instantiate it ONLY if either `VERILATOR` or `UVM` macro is defined.
```

Comment:
Nit: I think you need to switch these sentences over (because the define thing is to ensure the module isn't synthesized)